### PR TITLE
add slsa verifier installer action

### PR DESCRIPTION
**Pull request by mchmarny**
Replaces manual SLSA verifier download with the installer action

```
slsa-framework/slsa-verifier/actions/installer@v2.0.1
```

### CODE REVIEW

**Review by developer-guy — DISMISSED** [review]
lgtm, thanks

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -71,24 +71,13 @@ jobs:
   verification:
     needs: [goreleaser, provenance]
     runs-on: ubuntu-latest
+    env:
+      ATT_FILE_NAME: "attestation.intoto.jsonl"
```
Comment (developer-guy) [inline .github/workflows/release.yml:75]: is this `attestation.intoto` or `multiple`?

Comment (mchmarny) [reply .github/workflows/release.yml:75]: I'm not sure if multiples are fully supported yet but it works for me with the latest version [here](https://github.com/mchmarny/vulctl/blob/07e5cf2966f7629af3373bda0d3d8238dcd0c6f5/.github/workflows/on-tag-cli.yaml#L108-L114). In that case, the attestation file generated by SLSA Generator using `goreleaser` was `multiple.intoto.jsonl`

Comment (developer-guy) [reply .github/workflows/release.yml:75]: here is the PR that fixes this problem: https://github.com/ko-build/ko/pull/983/files

you may rebase from main

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -94,25 +94,14 @@ jobs:
       - provenance
 
     runs-on: ubuntu-latest
+    env:
+      ATT_FILE_NAME: "attestation.intoto.jsonl"
```
Comment (developer-guy) [inline .github/workflows/release.yml:98]: we don't need this anymore, you can remove this.

**Review by imjasonh — DISMISSED** [review]

**Review by developer-guy — DISMISSED** [review]

**Comment by developer-guy** [comment]
can you bump the version of the slsa-verifier-installer to 2.3.0 @mchmarny?

**Comment by mchmarny** [comment]
Done, updated to v2.3.0

**Comment by developer-guy** [comment]
@imjasonh we can merge this one.

**Review by developer-guy — APPROVED** [review]

**Pull request closed without merging**